pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -622,6 +623,9 @@ func (cf *changeFrontier) closeMetrics() {
 	// Delete this feed from the MaxBehindNanos metric so it's no longer
 	// considered by the gauge.
 	cf.metrics.mu.Lock()
+	if cf.metricsID > 0 {
+		cf.metrics.Running.Dec(1)
```

Comment:
huh, how does this play with node crashes?

Reply:
I think it plays well. `Running` is a gauge, not a counter, so when a node crashes it goes to zero for that node.